packages/components/src/custom-gradient-picker/gradient-bar/control-points.tsx

```diff
@@ -108,16 +122,28 @@ function ControlPoints( {
 	onStartControlPointChange,
 	onStopControlPointChange,
 	__experimentalIsRenderedInSidebar,
-} ) {
-	const controlPointMoveState = useRef();
+}: ControlPointsProps ) {
+	const controlPointMoveState: React.MutableRefObject<
+		ControlPointMoveState | undefined
+	> = useRef();
 
-	const onMouseMove = ( event ) => {
+	const onMouseMove = ( event: MouseEvent ) => {
 		const relativePosition = getHorizontalRelativeGradientPosition(
 			event.clientX,
 			gradientPickerDomRef.current
 		);
+
 		const { initialPosition, index, significantMoveHappened } =
-			controlPointMoveState.current;
+			controlPointMoveState?.current ?? {};
```

Comment:
We are guaranteed that `controlPointMoveState` will always be defined

```suggestion
			controlPointMoveState.current ?? {};
```